stenotype/packets.cc

```diff
@@ -306,6 +309,35 @@ Error PacketsV3::Builder::SetFanout(uint16_t fanout_type, uint16_t fanout_id) {
   return SUCCESS;
 }
 
+Error PacketsV3::Builder::SetPromisc() {
+  struct ifreq ifopts;
+  memset(&ifopts, 0, sizeof(ifopts));
+
+  VLOG(1) << "Setting promiscuous mode for " << state_.iface;
+	strncpy(ifopts.ifr_name, state_.iface, IFNAMSIZ-1);
```

Comment:
Some weird indenting here, maybe a tabs/spaces issue?  Could you reset the indent on lines 317-320 and 331-334 to 2 spaces?